diff --git a/node_modules/pdf-parse/index.js b/node_modules/pdf-parse/index.js
index 79c3623..822743a 100644
--- a/node_modules/pdf-parse/index.js
+++ b/node_modules/pdf-parse/index.js
@@ -7,7 +7,7 @@
 'use strict';
 
 const { readFileSync, promises: { readFile } } = require('fs');
-const { getDocument, PDFDataRangeTransport } = require('pdfjs-dist/legacy/build/pdf.js');
+const { getDocument, PDFDataRangeTransport } = require('pdfjs-dist/build/pdf.js');
 
 const DEFAULT_OPTIONS = {
     // internal page parser callback

